bentoml_cli/server/runner.py

```diff
@@ -29,6 +28,13 @@
     default=None,
     help="If set, start the server as a bare worker with the given worker ID. Otherwise start a standalone server with a supervisor process.",
 )
+@click.option(
+    "--worker-env-map",
+    required=False,
+    type=click.STRING,
+    default=None,
+    help="The environment variables to pass to the worker process. The format is a JSON string, e.g. '{0: {\"CUDA_VISIBLE_DEVICES\": 0}}'.",
```

Comment:
should we use a dotenv file instead of JSON?  that seems easier for debugging purpose

Reply:
The env map includes all envvars for each worker.
It seems not that easy to be represented with dotenv files.

Reply:
Agree that passing JSON in CLI isn't the most intuitive. Using an `.env` file, we can allow multiple arguments of key-value pairs.
```
--worker-env 0:worker_0.env
```

Reply:
@ssheng I think that's too complicated. This is not the public API.
The public API is the `bentoml serve` and `bentoml.serve`